client/blocks/login/two-factor-authentication/two-factor-actions.jsx

```diff
@@ -92,28 +95,47 @@ class TwoFactorActions extends Component {
 
 		return (
 			<Fragment>
-				{ this.props.isWoo && <FormDivider /> }
-				<Card className="two-factor-authentication__actions wp-login__links">
+				{ isWoo && twoFactorAuthType !== 'push' && <FormDivider /> }
```

Comment:
Fixes the double divider on the current `login/authenticator` screen

| Before | After |
|-|-|
| ![wordpress com_log-in_client_id=50916 redirect_to=https%3A%2F%2Fpublic-api wordpress com%2Foauth2%2Fauthorize%2F%3Fresponse_type%3Dcode%26client_id%3D50916%26state%3D1c22dc5ce0975240c46902a714d400aa844e01ee0e530cfbfd8a6890603fa7cc%26redirect](https://github.com/user-attachments/assets/7bf8766b-552f-4642-b4ad-e8fcb93647ce) | ![calypso localhost_3000_log-in_redirect_to=https%3A%2F%2Fpublic-api wordpress com%2Foauth2%2Fauthorize%2F%3Fresponse_type%3Dcode%26client_id%3D50916%26state%3D69c7f83c44d9438e013f60308e0d8369725e57fa68d18b558d8b41743976985b%26redirect_ur (8)](https://github.com/user-attachments/assets/07f73860-5e64-4f14-8a35-8f2b05c03c94) |